conans/model/requires.py

```diff
@@ -2,7 +2,7 @@
 
 from conans.errors import ConanException
 from conans.model.pkg_type import PackageType
-from conans.model.ref import ConanFileReference
+from conans.model.recipe_ref import RecipeReference
```

Comment:
This is something that I want to differentiate, between a ``RecipeReference``, which is exact, and a ``RequireReference`` which can contain a version range.